css/devcenter.scss

```diff
@@ -5,6 +5,7 @@
 @import "../synapse/code";
 @import "../synapse/variables";
 @import "../synapse/typography-modifiers";
+@import "../synapse/article";
```

Comment:
This is a temporary addition, will remove once the corresponding synapse PR is merged and built